packages/seacas/libraries/exodus/CMakeLists.txt

```diff
@@ -35,6 +35,12 @@ TRIBITS_ADD_LIBRARY(
 )
 
 set_property(TARGET exodus PROPERTY C_STANDARD 99)
+TARGET_INCLUDE_DIRECTORIES(
```

Comment:
Why were these added?  I think the include files have been getting installed without these...

Reply:
See https://github.com/gsjaardema/seacas/pull/214#issuecomment-776156905